web_responsive/static/src/css/web_responsive.scss

```diff
@@ -372,6 +382,10 @@ html .o_web_client .o_main .o_main_content {
                 max-width: 100%;
             }
 
+            .oe_title {
+                width: 100% !important;
```

Comment:
It seems like this `!important` should stay, however. Unless, maybe, if we switch to top full button row as suggested above.